content/en/security_monitoring/detection_rules/security_monitoring.md

```diff
@@ -0,0 +1,159 @@
+
+In this example, when greater than 5 failed logins and a successful login exist for the same `@usr.name`, the first case is matched and a Security Signal is generated.
+
+{{< img src="security_monitoring/detection_rules/gbv2.png" alt="Set rule cases" >}}
+
+## Set a Rule Case
+
+{{< tabs >}}
+{{% tab "Threshold" %}}
+
+### Trigger
+
+{{< img src="security_monitoring/detection_rules/define_rule_case.png" alt="Define the rule case" >}}
+
+Rule cases, such as `a > 3`, are evaluated as case statements. Thus, the first case to match generates the signal. Click and drag your rule cases to manipulate their ordering.
+
+A rule case contains logical operations (`>, >=, &&, ||`) to determine if a signal should be generated based on the event counts in the previously defined queries. The ASCII lowercase [query labels](#define-a-search-query) are referenced in this section.
+
+**Note**: The query label must precede the operator. For example, `a < 3` is allowed; `3 > a` is not allowed.
+
+Provide a **name**, for example "Case 1", for each rule case. This name is appended to the rule name when a signal is generated.
+
+### Severity and notification
+
+Set the severity of the Security Signal. The dropdown allows you to select an appropriate severity level (`INFO`, `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`).
+
+In the “Notify” section, configure zero or more [notification targets][1] for each rule case.
+
+### Time windows
+
+An `evaluation window` is specified to match when at least one of the cases matches true. This is a sliding window and evaluates in real time.
+
+Once a signal is generated, the signal will remain “open” if a case is matched at least once within this `keep alive` window. Each time a new event matches any of the cases, the *last updated* timestamp is updated for the signal.
+
+A signal will “close” regardless of the query being matched once the time exceeds the `maximum signal duration`. This time is calculated from the first seen timestamp.
+
+Additional cases can be added by clicking the **Add Case** button.
+
+**Note**: The `evaluation window` must be less than or equal to the `keep alive` and `maximum signal duration`.
+
+
+[1]: /monitors/notifications/?tab=is_alert#integrations
+{{% /tab %}}
+
+{{% tab "New Term" %}}
+
+{{< img src="security_monitoring/detection_rules/new_term_rule_case.png" alt="Define the rule case" >}}
+
+### Severity and notification
+
+Set the severity of the Security Signal. The dropdown allows you to select an appropriate severity level (`INFO`, `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`).
+
+In the “Notify” section, configure zero or more [notification targets][1] for each rule case.
+
+### Time windows
+
+To forget a value if it is not seen over a period of time, select a time window from the drop down menu.
+
+To keep updating the same signal if new values are detected, select the time window for if a signal is detected and the maximum duration you wish to keep updating the signal.
```

Comment:
needs a little copy edit here. we also should point out that if you want a unique signal for each and every new value, you want to configure this value to `0`.